CMakeLists.txt

```diff
@@ -126,6 +126,9 @@ endif()
 
 if (GODOT_ENABLE_HOT_RELOAD)
     set(GODOT_COMPILE_FLAGS "${GODOT_COMPILE_FLAGS} -D HOT_RELOAD_ENABLED")
+	if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
```

Comment:
We have the `compiler_is_gnu` variable defined above. Could we use?

Reply:
Yep! But since it's a generator expression, it requires a slightly different approach (using the target_compile_options function instead).